### PR TITLE
doc: Add fixed issues in v0.2 release note

### PR DESCRIPTION
**Pull request by NanlinXie**
Filter v0.2 fixed issues from github closed issue list.

Signed-off-by: Nanlin Xie <nanlin.xie@intel.com>

### CODE REVIEW

File: doc/release_notes_0.2.rst [hunk 1]
```diff
@@ -17,6 +17,21 @@ ACRN device model, and documentation.
 Version 0.2 new features
 ************************
 
+Fixed Issues
+============
+:acrn-issue:`663` - Black screen displayed after booting SOS/UOS
```
Comment (dbkinder) [inline doc/release_notes_0.2.rst]: The intention was to also add a description of the issue and if there is a workaround (see the 0.1 release notes for an example).

Each of these issues needs to have a blank line between them (or they'll run together as one paragraph).  (Click on the VIEW button here in GitHub to see what I mean.)

Comment (dbkinder) [reply doc/release_notes_0.2.rst]: The VIEW button when you're looking at the file changed:  https://github.com/projectacrn/acrn-hypervisor/blob/7fc0fd8bd1f12328bb492ff0f3e891656462f21a/doc/release_notes_0.2.rst

**Comment by NanlinXie** [comment]
@dbkinder, These are for fixed issues which need not workaround, should we still need short description for them?

File: doc/release_notes_0.2.rst (hunk 1 above)
Comment (dbkinder) [reply doc/release_notes_0.2.rst]: doh.  You're right, I was thinking these are Known issues not fixed ones.

**Pull request merged**